network/p2p/libp2pNode.go

```diff
@@ -113,6 +114,9 @@ func NewLibP2PNode(logger zerolog.Logger,
 		return nil, fmt.Errorf("could not bootstrap libp2p host: %w", err)
 	}
 
+	pingLibP2PProtocolID := generateProtocolID(rootBlockID)
+	pingService := NewPingService(libP2PHost, pingLibP2PProtocolID, logger)
```

Comment:
create the PingService to handle incoming ping request and to send ping request to remote nodes